packages/reactivity/src/reactive.ts

```diff
@@ -217,13 +278,26 @@ function createReactiveObject(
   return proxy
 }
 
+/**
+ * Checks if an object is a reactive proxy created by `reactive`.
```

Comment:
```suggestion
 * Checks if an object is a proxy created by `reactive` or `shallowReactive`.
```

Reply:
Although I think we should make this sentence more generic instead as [it works in more scenarios than just these 2](https://sfc.vuejs.org/#eNqNUktuwyAQvcqITRLJMXvLitQrdM3GdaEhMh8BdlQh7t7BSRy3QVZ3zOP9BBPJm7X1NHLSkNb3TtoAnofRnpiWyhoXIILjogLp33nXBznxCoHV6dPo4bsCf+6GwVyR9ALcuQsgIIFwRsEOg3dMt/SWjJk4BK7s0AWOE0BrHT894hqIcdVj/8D3MR0OkFJLM3stm6ssdQv6mfDbqOgkXrUis+upG0a+HV7QLrmzyZaLAOukkuX6Yh8c6grC51v/VT1vNvv/w6D8+C1dPpBU5LZDR9XZ+uKNxi2LOYLdLzwjaJ+RjOE25JmRcwjWN5R60efdvPjauC+Kp9qNOkjFa+7V8cOZq+cOjRnJFonpRNIPlAwBdg==).

I would say _if a value is a Vue-created proxy of an object_ but let's wait on more feedback

Reply:
Hmm, I think sticking with the docs here makes sense. _But maybe the docs would need some beefing up regarding the usage in the playground, no?_